app/qml/PositionMarker.qml

```diff
@@ -2,16 +2,25 @@ import QtQuick 2.3
 import QtQuick.Controls 2.2
 import QtQml 2.2
 import QtGraphicalEffects 1.0
+import QtSensors 5.11
 import QgsQuick 0.1 as QgsQuick
 import "."
+import lc 1.0
 
 Item {
     id: positionMarker
     property int size: InputStyle.rowHeightHeader/2
     property QgsQuick.PositionKit positionKit
     property color baseColor: InputStyle.highlightColor
     property bool withAccuracy: true
+    property real direction: -1
 
+    onPositionKitChanged: positionDirection.positionKit = positionMarker.positionKit
+
+    PositionDirection {
+      id: positionDirection
+      onDirectionChanged: direction.rotation = positionDirection.direction
```

Comment:
Is this needed? the "direction" item has "rotation" set to positionDirection.direction below